_sass/components/_svg.scss

```diff
@@ -3,8 +3,8 @@
 
 .svg-wrapper {
   svg {
-    height: $icon-width;
-    width: $icon-width;
+    height: 0.8rem;
+    width: 0.8rem;
```

Comment:
should this be derived from the font family like the line heights? or is it a separate calculation?

Reply:
I don't think I put much thought into this other than duplicating the existing value previously assigned, since it started erroring because USWDS now has a variable with the same name.

I'll poke at it and see what it's used for and how we'd want to deal with it.

Reply:
Removing the style separately in #1221